Editor/Scripts/interactivetutorials_dialog.py

```diff
@@ -124,6 +129,9 @@ def __init__(self, parent=None):
         self.content_area.setWordWrap(True)
         self.tutorial_layout.addWidget(self.content_area, 1)
 
+        self.step_label = QLabel(self)
+        self.tutorial_layout.addWidget(self.step_label)
```

Comment:
I think these lines and several below are supposed to be in the separate step counter PR https://github.com/cgalvan/InteractiveTutorials/pull/26

Reply:
Yes, I can clean them out.